documentation/CAMARA-Security-Interoperability.md

```diff
@@ -0,0 +1,295 @@
+Therefore scopes should be available to API implementations.
+
+---
+
+
+## Missing "openid" scope
+
+[OIDC Core Authentication Request](https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest) states the following about the value of scope.
+> REQUIRED. OpenID Connect requests MUST contain the openid scope value. If the openid scope value is not present, the behavior is entirely unspecified. Other scope values MAY be present. ...
+
+This document defines that the authentication server SHOULD not return an id token if `openid` is missing in the scope parameter.
+
+This document defines the following error handling for a missing "openid" value in scope. 
+Please refer to [Authentication Error Response](https://openid.net/specs/openid-connect-core-1_0.html#AuthError).
+
+If "openid" is missing in the scope value but a claim that is [standardized in OIDC](https://openid.net/specs/openid-connect-core-1_0.html#Claims) is requested, then the Authorization Server returns an HTTP response code of 400 (Bad Request) and an error invalid_request.
+
+Clients SHOULD follow the OIDC standard and SHOULD include `openid` in the list of requested scopes.
+Without id token there is no `sub` field and the privacy features of OIDC are severely crippled. 
+Globally unique identifiers, like the MSISDN, should be avoided for privacy reasons.
+
+## Purpose
+
+In Camara `purpose` is used to convey to the user for which purpose an API is used. E.g.: Does the client request the user's location because it wants to show relevant tourist information or is the user's location informat protecting the user when they withdraw money from an ATM?
+Therefore `purpose`is only used in Camara when user consent is required (CIBA or OIDC authorization code flow) and a potential `prompt` parameter's value is NOT `none`.
+
+
+### Purpose as a scope
+
+Purpose is one of the scope parameter's values. There MUST be exactly one purpose.
+
+The Authorization Server identifies the purpose using the prefix `dpv:`.
+
+This scope MUST have the following format: `dpv:<dpvValue>` where `<dpvValue>` is coming from [W3C DPV purpose definition](https://w3c.github.io/dpv/dpv/#vocab-purpose)
+
+
+
+### Outlook on purpose-handling leveraging Rich Authorization Request
+
+[RFC 9396 OAuth 2.0 Rich Authorization Requests](https://www.rfc-editor.org/rfc/rfc9396) defines a OAuth2 parameter `authorization_details` that is used to carry fine-grained authorization data. The parameter `authorization_details` can be used in all places where OAuth2 allows the `scope` parameter. That means that the parameter `authorization_details` can be used in all places where OIDC and CIBA allow the `scope` parameter.
+
+This document defines exactly one format to be used in Camara for the value of the `authorization_details` parameter that is used additionally to the scope parameter.
```

Comment:
@AxelNennker @Elisabeth-Ericsson IMHO, we better not document the RAR option in this profile release version. We may have a separate issue to reflect the agreement reached to consider the use of RAR for future releases when more complex scenarios/requirements may be needed (and include the proposed `authorization_details` format there). But documenting an unfinished solution now may be misleading. I would simplify the current profile version to just document the purpose management solution that applies now.

Reply:
As Elisabeth said during the interim meeting yesterday the outlook tells implementers what this groups seems likely to come. I think that is helpful and not misleading. I got the impression yesterday that there was broad agreement that RAR is the way to go.

Reply:
No worries.
Putting myself into the shoes of a developer to whom somebody says "Implement an AZ that adheres to Camara", I would read this security and interoperability document. But getting RAR implemented needs time and people probably should start thinking in that direction now. Because otherwise when we require RAR and people start working on RAR then, the whole process takes month/years.